superset/config.py

```diff
@@ -282,8 +282,8 @@
 # in the results backend. This also becomes the limit when exporting CSVs
 SQL_MAX_ROW = 100000
 
-# Limit to be returned to the frontend.
-DISPLAY_MAX_ROW = 1000
+# Default row limit for SQL Lab queries
+DEFAULT_SQLLAB_LIMIT = 10000
```

Comment:
@mistercrunch Sql Lab used to only render 1000 rows for each query, but fetch a much bigger number of rows of results. Do you have any suggestion on this default query limit number?